src/plugins/content_management/common/rpc/create.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import { schema } from '@kbn/config-schema';
+
+import type { ProcedureSchemas } from './types';
+
+export const createSchemas: ProcedureSchemas = {
+  in: schema.object(
+    {
+      contentTypeId: schema.string(),
+      // --> "data" to create a content will be defined by each content type
+      data: schema.recordOf(schema.string(), schema.any()),
+      options: schema.maybe(schema.object({}, { unknowns: 'allow' })),
+    },
+    { unknowns: 'forbid' }
+  ),
+  out: schema.maybe(schema.object({}, { unknowns: 'allow' })),
+};
+
+export interface CreateIn<
+  T extends string = string,
+  Data extends object = Record<string, unknown>,
```

Comment:
```suggestion
  Data extends object = object,
```

Reply:
From my experiments in https://github.com/elastic/kibana/pull/151163 that we discussed previously this seemed to be better for users. Same for other types

Reply:
Done in https://github.com/elastic/kibana/pull/151264/commits/fb329b0ce7692c54a87c4eb8cbf3a08f7271e826